data.sql

```diff
@@ -7,3 +7,44 @@ VALUES ('Agumon', '2020-02-03', 0, true, 10.23),
 ('Gabumon', '2018-11-15', 2, true, 8),
 ('Pikachu', '2021-01-07', 1, false, 15.04),
 ('Devimon', '2017-05-12', 5, true, 11);
+
+-- 
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES 
+('Charmander', '2020-02-08', 0, false, -11),
+('Plantmon', '2022-11-15', 2, true, -5.7),
+('Squirtle', '1993-04-02', 3, false, -12.13),
+('Angemon', '2005-07-12', 1, true, -45),
+('Boarmon', '2005-06-07', 7, true, 20.4),
+('Blossom', '1998-10-13', 3, true, 17);
+
+BEGIN;
+UPDATE animals 
+SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals 
+SET species = 'Digimon'
+WHERE NAME LIKE '%mon';
+UPDATE animals
+SET species = 'Pokemon'
+WHERE NAME NOT LIKE '%mon';
+COMMIT;
```

Comment:
* Kindly Verify that change was made and persists after commit. as stated in the project requirement [Check here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made%20and%20persists%20after%20commit.)
* HINT 💭 use the above ⬆️  comment as a guide